src/js/bootstrap-slider.js

```diff
@@ -1500,9 +1500,7 @@
 				this.$sliderElem.off();
 			},
 			_setText: function(element, text) {
-				if(typeof element.innerText !== "undefined") {
-			 		element.innerText = text;
-			 	} else if(typeof element.textContent !== "undefined") {
+				if(typeof element.textContent !== "undefined") {
```

Comment:
instead of removing the `innerText` check, could you just swap textContent and innerText in the original code?

That way, textContent will be the default API for setting the text...which is probably what we should be doing [based on this article](http://www.kellegous.com/j/2013/02/27/innertext-vs-textcontent/)